### DISCUSSION TITLE
NOTAM uplink from simbrief

### DISCUSSION THREAD

**Discussion by OldGroof** [magknight/787aviatorsTracker#521] [Ideas]
**Is your feature request related to a problem? Please describe.**
Not related to a problem

**Describe the solution you'd like**
On the performance pages of the EFB there is a notams button. Have that selectable and have it show the list of notams that simbrief gives you on the OFP

**Describe alternatives you've considered**
Not sure what the alternatives would be.

**Comment by aeroniemi** [comment]
need to find a way that is actually sensible to display these - unsure at present

**Comment by aeroniemi** [comment]
the ones you find in the EFB are company-generated things associated with specific airport/runways - we couldn't do that as decoding them programatically is effectively impossible
we could add a way to request them through comm, although that requires UI design as the real aircraft can't do that
i did put some thinking into that, but i don't really have a good enough solution to actually write it
so, if you can come up with a user story and UI for it, feel free to add it to the discussion
